test/Ownable.py

```diff
@@ -14,22 +14,30 @@ def event_loop():
 @pytest.fixture(scope='module')
 async def ownable_factory():
     starknet = await Starknet.empty()
-    owner = await starknet.deploy("contracts/Account.cairo")
-    ownable = await starknet.deploy("contracts/Ownable.cairo")
-    await owner.initialize(signer.public_key, owner.contract_address).invoke()
-    await ownable.initialize_ownable(owner.contract_address).invoke()
+    owner = await starknet.deploy(
+        source="contracts/Account.cairo",
+        constructor_calldata=[signer.public_key]
+    )
```

Comment:
does it work without the explicit parameters? something like

```python
owner = await starknet.deploy("contracts/Account.cairo", [signer.public_key])
```

Reply:
It returns an assertion error without the keyword arguments. See #47 . Let me know what you think